### DISCUSSION TITLE
"/yarn.lock": not found

### DISCUSSION THREAD

**Discussion by plauge** [FlowiseAI/Flowise#113] [Q&A] [answered]
Hi,

I am a beginner and trying to set up Flowice on render.com by following this instruction: https://www.youtube.com/watch?v=EsI_7L0fzKk&lc=UgwcfB3tRijVVb3Q-yp4AaABAg.9pjCkzAGISM9pjrM21WW1r

What am I doing wrong? :-)

Here is my error log:

May 16 05:40:12 AM  ==> Cloning from https://github.com/plauge/FlowiseAI...
May 16 05:40:12 AM  ==> Checking out commit 1d9181f099e4cdaf6137e329d7a1c014741b81fd in branch main
May 16 05:40:15 AM  #1 [internal] load .dockerignore
May 16 05:40:15 AM  #1 transferring context: 98B done
May 16 05:40:15 AM  #1 DONE 0.0s
May 16 05:40:15 AM  
May 16 05:40:15 AM  #2 [internal] load build definition from Dockerfile
May 16 05:40:15 AM  #2 transferring dockerfile: 661B done
May 16 05:40:15 AM  #2 DONE 0.0s
May 16 05:40:15 AM  
May 16 05:40:15 AM  #3 [internal] load metadata for docker.io/library/node:18-alpine
May 16 05:40:15 AM  #3 ...
May 16 05:40:15 AM  
May 16 05:40:15 AM  #4 [auth] library/node:pull token for registry-1.docker.io
May 16 05:40:15 AM  #4 DONE 0.0s
May 16 05:40:15 AM  
May 16 05:40:15 AM  #3 [internal] load metadata for docker.io/library/node:18-alpine
May 16 05:40:16 AM  #3 DONE 1.2s
May 16 05:40:16 AM  
May 16 05:40:16 AM  #5 [ 1/10] FROM docker.io/library/node:18-alpine@sha256:1ccc70acda680aa4ba47f53e7c40b2d4d6892de74817128e0662d32647dd7f4d
May 16 05:40:16 AM  #5 resolve docker.io/library/node:18-alpine@sha256:1ccc70acda680aa4ba47f53e7c40b2d4d6892de74817128e0662d32647dd7f4d done
May 16 05:40:16 AM  #5 ...
May 16 05:40:16 AM  
May 16 05:40:16 AM  #6 [internal] load build context
May 16 05:40:16 AM  #6 transferring context: 6.78MB 0.2s done
May 16 05:40:16 AM  #6 DONE 0.2s
May 16 05:40:16 AM  
May 16 05:40:16 AM  #7 [ 3/10] COPY package.json ./
May 16 05:40:16 AM  #7 CACHED
May 16 05:40:16 AM  
May 16 05:40:16 AM  #8 [ 4/10] COPY yarn.lock ./
May 16 05:40:16 AM  #8 ERROR: failed to calculate checksum of ref osbee4tb8zaiurnjtchndk2nq::1shxoliizx9bupf50q1pu0ak8: "/yarn.lock": not found
May 16 05:40:16 AM  
May 16 05:40:16 AM  #9 [ 2/10] WORKDIR /usr/src/packages
May 16 05:40:16 AM  #9 CANCELED
May 16 05:40:16 AM  ------
May 16 05:40:16 AM   > [ 4/10] COPY yarn.lock ./:
May 16 05:40:16 AM  ------
May 16 05:40:16 AM  Dockerfile:11
May 16 05:40:16 AM  --------------------
May 16 05:40:16 AM     9 |     # Copy root package.json and lockfile
May 16 05:40:16 AM    10 |     COPY package.json ./
May 16 05:40:16 AM    11 | >>> COPY yarn.lock ./
May 16 05:40:16 AM    12 |
May 16 05:40:16 AM    13 |     # Copy components package.json
May 16 05:40:16 AM  --------------------
May 16 05:40:16 AM  error: failed to solve: failed to compute cache key: failed to calculate checksum of ref osbee4tb8zaiurnjtchndk2nq::1shxoliizx9bupf50q1pu0ak8: "/yarn.lock": not found
May 16 05:40:16 AM  error: exit status 1

**Comment by cryptskii** [comment]
I am also having the same problem

**Reply by cryptskii** [reply]
I’m wondering if this has anything to do with why I can’t write files to my local Drive using the write file addon

**Comment by plauge** [comment]
I found this way to do it - will test it tomorrow:
https://youtu.be/s33v5cIeqA4?t=120

**Reply by plauge** [reply, marked as answer]
This way works for me: https://youtu.be/s33v5cIeqA4?t=120
